README.md

```diff
@@ -38,6 +38,8 @@ cds compile <path to srv folder> --to ord [-o] [destinationFilePath]
 1) Run `cds watch` in the application's root.
 2) Check the following relative paths for ORD information - `/.well-known/open-resource-discovery` , `/open-resource-discovery/v1/documents/1`.
 
+![](ord.gif)
```

Comment:
you should use image tag like [this](https://github.com/cap-js/notifications/blob/main/README.md?plain=1#L114)